cli/pack/app_dir.go

```diff
@@ -0,0 +1,24 @@
+package pack
+
+import (
+	"github.com/otiai10/copy"
+	"os"
+	"path/filepath"
+)
+
+// copyBundleDir copies a packed bundle content to the passed package
+// directory with creating a passed prefix.
+func copyBundleDir(packagePath, bundlePath, rootPrefix string) error {
+	packageDebDir := filepath.Join(packagePath, rootPrefix)
```

Comment:
How does this function know that passed packagePath is a debian package?.. Is this Debian specific API? Function and parameter names do not reflect this.